built/end_session_redirect_based_handler.d.ts

```diff
@@ -0,0 +1,22 @@
+import { AuthorizationServiceConfiguration } from './authorization_service_configuration';
+import { RandomGenerator } from './crypto_utils';
+import { EndSessionRequest } from './end_session_request';
+import { EndSessionRequestHandler, EndSessionRequestResponse } from './end_session_request_handler';
+import { StorageBackend } from './index';
+import { BasicQueryStringUtils } from './query_string_utils';
+import { LocationLike } from './types';
+/**
+ * Represents an EndSessionRequestHandler which uses a standard
+ * redirect based code flow.
+ */
+export declare class EndSessionRedirectRequestHandler extends EndSessionRequestHandler {
+    storageBackend: StorageBackend;
+    locationLike: LocationLike;
+    constructor(storageBackend?: StorageBackend, utils?: BasicQueryStringUtils, locationLike?: LocationLike, generateRandom?: RandomGenerator);
+    performEndSessionRequest(configuration: AuthorizationServiceConfiguration, request: EndSessionRequest): void;
+    /**
+     * Attempts to introspect the contents of storage backend and completes the
+     * request.
```

Comment:
can add the "request" part to the previous line.

Reply:
done